boards/arm64/mimx93_evk/mimx93_evk-pinctrl.dtsi

```diff
@@ -9,8 +9,8 @@
 &pinctrl {
 	uart2_default: uart2_default {
 		group0 {
-			pinmux = <&iomuxc_uart2_rxd_uart_rx_uart2_rx>,
-				<&iomuxc_uart2_txd_uart_tx_uart2_tx>;
+			pinmux = <&iomuxc1_uart2_rxd_lpuart_rx_lpuart2_rx>,
```

Comment:
Please add a commit updating `west.yml` to reference https://github.com/zephyrproject-rtos/hal_nxp/pull/273, since you are updating the pin control node names here.

Reply:
As https://github.com/zephyrproject-rtos/hal_nxp/pull/273 depends on this commit, you can create a seperate PR for this patch in order to make thing simple.